README.md

```diff
@@ -46,7 +47,7 @@ Currently, we support the following data types:
 The `Any` module contains options that will be common to all data types. It supports the following options:
   * *Required*
 
-  Allowed values are boolean `true` and `false`. The default value of `required` is set to `false`.
+  Marks a parameter as required which will not allow `nil` as value. Allowed values are boolean `true` and `false`. The default value of `required` is set to `false`.
```

Comment:
Setting `required` to `true` will cause a validation error when a field is not present or the value is `nil`. Allowed values for `required` are `true` and `false`. The default is `false`.